### PR TITLE
DOC-464: add examples to shortcut api docs

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-464

Description of Changes:
* Add useful examples to `editor.shortcuts.add` and `editor.addshortcut`API docs

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)
* [x] License headers added on new files (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/core/main/ts/api/Editor.ts [hunk 1]
```diff
@@ -540,6 +540,23 @@ class Editor implements EditorObservable {
    * @param {String/Function} cmdFunc Command name string or function to execute when the key is pressed.
    * @param {Object} scope Optional scope to execute the function in.
    * @return {Boolean} true/false state if the shortcut was added or not.
+   * @example
+   * editor.addShortcut('ctrl+a', "description of the shortcut", function() {});
+   * editor.addShortcut('ctrl+alt+a', "description of the shortcut", function() {});
+   * editor.addShortcut('meta+a', "description of the shortcut", function() {});
+   * // "meta" maps to Command on Mac and Ctrl on PC
```
Comment (metricjs) [inline modules/tinymce/src/core/main/ts/api/Editor.ts:547]: having the comment on a new line after the line it refers to reads oddly to me? I may just used to comments being before/inline though

Comment (lnewson) [reply modules/tinymce/src/core/main/ts/api/Editor.ts:547]: Agreed... I thought this applied to the next line to be honest. These comments should be above the line, as that's the normal convention.

File: modules/tinymce/src/core/main/ts/api/Editor.ts [hunk 2]
```diff
@@ -540,6 +540,23 @@ class Editor implements EditorObservable {
...
+   * editor.addShortcut('access+a', "description of the shortcut", function() {});
+   * // "access" maps to Control+Option on Mac and shift+alt on PC
+   *
+   * editor.addShortcut(
+   *  'meta+access+c', 'Opens the code editor dialog.', function () {
```
Comment (lnewson) [inline modules/tinymce/src/core/main/ts/api/Editor.ts:552]: Why was this added on a new line? It's not too bad but it is a little odd, so would be better on a single line (likewise for the other examples).

Comment (tylerkelly13) [reply modules/tinymce/src/core/main/ts/api/Editor.ts:552]: Unfortunately the CCS on the API docs is pretty messy. Overflowing content is hidden, not wrapped.

File: modules/tinymce/src/core/main/ts/api/Editor.ts (hunk 1 above)
Comment (tylerkelly13) [reply modules/tinymce/src/core/main/ts/api/Editor.ts:546]: Fixed by https://github.com/tinymce/tinymce/pull/5818/commits/8b8bd26d364e6dade5085a758b2bf0484967a07b
